### ISSUE TITLE
AppStore won't let upload the binary using LiquidCore

### ISSUE THREAD

**Issue by dsemenovsky** [LiquidPlayer/LiquidCore#180]
ITMS-90338: Non-public API usage - The app references non-public symbols in Frameworks/LiquidCore.framework/LiquidCore: _getcontext, _makecontext, _setcontext. If method names in your source code match the private Apple APIs listed above, altering your method names will help prevent this app from being flagged in future submissions. In addition, note that one or more of the above APIs may be located in a static library that was included with your app. If so, they must be removed. For further information, visit the Technical Support Information at

**Comment by KGMaxey** [comment]
Can confirm this issue

**Comment by KGMaxey** [comment]
Update: as a workaround until this PR get's approved, adding this to your pod file appears to work. It's the same fix as the PR just post install.

```
post_install do |installer|
  installer.pods_project.targets.each do |target|
    if target.name == "LiquidCore"
      print "Found LiquidCore Target. Updating CFLAGS...\n"
      target.build_configurations.each do |config|
        cflags = config.build_settings['OTHER_CFLAGS'] || ['$(inherited)']
        cflags << '-DOPENSSL_NO_ASYNC'
        config.build_settings['OTHER_CFLAGS'] = cflags
      end
    end
  end
end
```

**Comment by ericwlange** [comment]
Thanks for the contribution!  This has been merged and released in 0.7.9.
